### PR TITLE
[#1404] DG: edit the running from code part

### LINKED ISSUE #1404: Update the documentation for running RepoSense from code
**What problem(s) did you run into that caused you to request additional documentation? What, if any, existing documentation relates to this proposal?**

I cannot view the RepoSense report generated in the browser following the Running the app from code section in the Developer Guide because some gradle instructions seem to be missing.

**What would you like added or modified in the documentation?**

I would like to propose to update the documentation so that the user can actually view the report in the browser from code by executing the gradle instruction.

### PR DESCRIPTION
**Pull request by HCY123902**
Resolves #1404 

Commit message
```
Edit the running from code section of the Developer Guide

The documentation does not specifically mention how to view the
report in the browser when running the application from code.

Adding the additional documentation can bring convenience for
newcomers to run the application locally when developing.

Let's edit the documentation to make the instructions clearer.
```

### CODE REVIEW

File: docs/dg/workflow.md [hunk 1]
```diff
@@ -37,9 +37,12 @@ Our workflow is mostly based on the guidelines given at se-education.org/guides.
 
 <div id="section-running-from-code">
 
-* To run the app from code, run `gradlew run` from the project root. By default, it will run based on the config files in the `[project root]/config` folder.
-* To supply flags, you can use the `-Dargs="[FLAGS]"` format.<br>
+* To run the app from code, run `gradlew run` from the project root. By default, it will run based on the config files in the `[project root]/config` folder, and generate the report in the `[project root]/reposense-report` folder.
+* To supply flags to customize the report, you can use the `-Dargs="[FLAGS]"` format.<br>
   e.g., `gradlew run -Dargs="--since 31/12/2019 --formats java adoc xml"`
+* Run `gradlew run -Dargs="--view ./reposense-report"` to view the generated report in the default browser.
```
[the remark this answers is not in the document]
Comment (HCY123902) [reply docs/dg/workflow.md]: Thanks for pointing it out. It is the default. I have edited it accordingly.

**Review by Tejas2805 — APPROVED** [review]
LGTM!

**Review by fzdy1914 — APPROVED** [review]

**Comment by fzdy1914** [comment]
The documentation does not specifically mention how to view the 
report in the browser when running the application from code.

Adding the additional documentation can bring convenience for 
newcomers to run the application locally when developing.

Let's edit the documentation to make the instructions clearer.

**Comment by Tejas2805** [comment]
@HCY123902 Can you update the commit message based on the comment by @fzdy1914 ?

**Comment by HCY123902** [comment]
Sure, no problem. Sorry I did not notice this previously.

**Pull request merged**